Tests/SwiftSyntaxBuilderTest/Assertions.swift

```diff
@@ -16,10 +16,10 @@ import _SwiftSyntaxTestSupport
 
 import XCTest
 
-func AssertBuildResult<T: SyntaxProtocol>(_ buildable: T, _ expectedResult: String, file: StaticString = #file, line: UInt = #line) {
+func AssertBuildResult<T: SyntaxProtocol>(_ buildable: T, _ expectedResult: String, trimTrailingWhitespace: Bool = true, file: StaticString = #file, line: UInt = #line) {
   AssertStringsEqualWithDiff(
```

Comment:
Instead of the ternary, what do you think about the following?

```swift
var buildableDescription = buildable.formatted().description
var expectedResult = expectedResult
if trimTrailingWhitespace {
  buildableDescription = buildableDescription.trimTrailingWhitespace()
  expectedResult = expectedResult.trimTrailingWhitespace()
}
```